.pre-commit-config.yaml

```diff
@@ -22,8 +22,8 @@ repos:
   rev: 22.3.0
   hooks:
   - id: black
-- repo: https://gitlab.com/pycqa/flake8
-  rev: 3.9.2
+- repo: https://github.com/pycqa/flake8
+  rev: 6.0.0
```

Comment:
@ogajduse Thanks for fixing this, I was about to raise a PR :)
Can you also run auto-update to update hooks `pre-commit autoupdate ; pre-commit run -a` ?

Reply:
Will do! Gimme a few.